src/java.xml/share/classes/javax/xml/stream/XMLOutputFactory.java

```diff
@@ -56,7 +56,7 @@
  * <p>The following paragraphs describe the namespace and prefix repair algorithm:
  *
  * <p>The property can be set with the following code line:
- * {@code setProperty("javax.xml.stream.isRepairingNamespaces", new Boolean(true|false));}
+ * {@code setProperty("javax.xml.stream.isRepairingNamespaces", Boolean.TRUE|Boolean.FALSE);}
```

Comment:
From a code standpoint I don't see why the OR operation is needed. Is to emphasize that true or false can be passed into the parameter?

Reply:
I think the later, that the value can be true or false. As an example, one value is fine, e.g.
  {@code setProperty("javax.xml.stream.isRepairingNamespaces", Boolean.TRUE);} 
  or 
  {@code setProperty("javax.xml.stream.isRepairingNamespaces", true);}
  let autoboxing handle it.

Reply:
The` true|false` should just be changed to just set the value to _true_ or _false_.

I think Boolean.TRUE is fine.   If the consensus is to use autoboxing in the example that is fine also.

Reply:
Thank you Joe and Lance, will replace it with a single value